### DISCUSSION TITLE
Getting the error "GUI resources are not identified by the URI" after calling RenderAsync()

### DISCUSSION THREAD

**Discussion by Ctrn** [soundaranbu/RazorTemplating#63] [Q&A]
I am encountering the following issue when using the Nuget package in WPF application. 

After calling the RenderAsync () method from .net5 WPF application; I got this error from other UI windows:
The component "ComponentName" does not have a resource identified by the URI "URI_Path_to_xaml_file"

It appears that the packages changes the references of the xaml after calling the Razor
Can you please support us with ideas why this could happen and if possible to overcome it; I ran out of ideas

**Comment by soundaranbu** [comment]
Hey @Ctrn, that's unfortunate. Can I ask what version of the library are you using? Have you tried to replicate the same issue in the latest versions of both .NET and the library?
